docs/course_materials/linux_intro.html

```diff
@@ -515,6 +515,34 @@ <h3><b>clear</b></h3>
           <section data-markdown="exercises/01_Linux/class_exercise_2.md" data-separator="^\n\n\n" data-separator-vertical="^\n\n">
           </section>
         </section>
+        <section>
+          <section>
+            <h3><b>Vim</b></h3>
+            <ul>
+              <li>Vim is an editor to create or edit a text file.</li>
+              <li>There are two modes in vim. One is the command mode and another is the insert mode.</li>
+              <ul>
+                <li>In the command mode, user can move around the file, delete text, etc.</li>
+                <li>In the insert mode, user can insert text.</li>
+              </ul>
+            </ul>
+          </section>
+          <section>
+            <h3><b>Usefull vim commands</b></h3>
+            <ul>
+              <li>i - Enter insert mode.</li>
+              <li>esc - Exit insert mode.</li>
```

Comment:
a few more useful things I think we should add:
jump to the end of the file, 
jump to the beginning of the file. 
search words
go to a specific line. 
@yifatmakias can you add those?

Reply:
Yes, I will add those :)